x/nft/client/testutil/grpc.go

```diff
@@ -0,0 +1,518 @@
+package testutil
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/testutil/rest"
```

Comment:
This is only in [v0.46.0-alpha1](https://pkg.go.dev/github.com/cosmos/cosmos-sdk@v0.46.0-alpha1?tab=versions) of the cosmos-sdk and we need to use v0.44.3 everywhere else, so this is a bit problematic.

Reply:
should be solved with the protocgen.sh that uses third_party/proto